xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/widgets/modalPopup.js

```diff
@@ -79,7 +79,7 @@ widgets.ModalPopup = Class.create({
     }
     // Add the close button
     if (this.options.displayCloseButton) {
-      var closeButton = new Element('div', {'class': 'xdialog-close', 'title': 'Close'}).update("&#215;");
+      var closeButton = new Element('button', {'class': 'btn btn-default xdialog-close', 'title': 'Close'}).update("&#215;");
```

Comment:
I know this hasn't been changed, but this "Close" title is supposed to be translated.